src/components/UpcomingContributors.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {
+    Grid
+} from '@material-ui/core'
+import ContributorTile from './ContributorTile'
+
+const UpcomingContributors = React.memo((props) => {
+    const {
+        contributors,
+        active,
+        project,
+        addAllocation
+    } = props
+
+    return (
+        contributors.map(c => {
+            return (
+                <Grid item xs={12} md={6} key={c.id}>
```

Comment:
Add the className 'UpcomingContributors' in here even if it doesn't have a sass file, this helps when debugging directly in the web inspector. As a personal preference, I like to return an element instead of a map function, so I would create a return with the item row an inside of it call a renderUpcomingContributors function with the map